app/views/liquids/_open_graph.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="crayons-card p-5">
+  <% if page.image %>
+    <div class="s:max-w-50 mb-4">
+      <img src="<%= page.image %>" height="auto" alt="Avatar">
+    </div>
+  <% end %>
+  <div class="color-primary">
+    <p class="fs-xl fw-bold lh-tight mb-2">
+      <%= page.title %>
+    </p>
+    <p class="truncate-at-2 mb-2">
+      <%= page.description %>
```

Comment:
Is there **always** going to be a description? If not, shouldn't we wrap the entire paragraph with similar IF statements as you have for image `if page.description`?

Reply:
Great question! I'm actually going to do more of that "checking" logic in the backend, and keep as much of it out of the view. 👍🏾